lib/resources.js

```diff
@@ -205,7 +221,28 @@ module.exports = function (app, _private = null) {
       }
     }
 
-    body = params.itemUri ? addItemQuery(body, params.itemUri) : addInnerHits(body, { size: params.items_size, from: params.items_from, merge_checkin_card_items: params.merge_checkin_card_items })
+    // Are we selecting a specific item?
+    if (params.itemUri) {
+      // Add that item uri as a term query to the ES body:
+      body = addItemQuery(body, params.itemUri)
+    } else {
+      // No specific item requested, so add pagination and matching params:
+      const itemsOptions = {
+        size: params.items_size,
+        from: params.items_from,
+        merge_checkin_card_items: params.merge_checkin_card_items,
+        query: {
+          volume: params.item_volume,
+          date: params.item_date,
+          format: params.item_format,
+          location: params.item_location,
+          status: params.item_status,
+          include_item_aggregations: params.include_item_aggregations
```

Comment:
Why include `include_item_aggregations` in `itemOptions.query`? Doesn't appear to be used in `addInnerHIts`